### PR TITLE
Add compat data for CSS

### PR DESCRIPTION
**Pull request by bunnybooboo**
https://developer.mozilla.org/en-US/docs/Web/API/CSS

### CODE REVIEW

File: api/CSS.json [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "22",
+              "notes": "Was available behind the <code>layout.css.supports-rule.enabled</code> preference since Gecko 20."
```
Comment (ExE-Boss) [inline api/CSS.json]: Firefox doesn’t support `paintWorklet`s.

File: api/CSS.json [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+            },
+            "firefox_android": {
+              "version_added": "22",
+              "notes": "Was available behind the <code>layout.css.supports-rule.enabled</code> preference since Gecko 20."
```
Comment (ExE-Boss) [inline api/CSS.json]: ditto

File: api/CSS.json [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+      },
+      "paintWorklet": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSS/paintWorklet",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": "28"
```
Comment (ExE-Boss) [inline api/CSS.json]: This is `65`

File: api/CSS.json [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": true
```
Comment (ExE-Boss) [inline api/CSS.json]: `false`

File: api/CSS.json [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+            },
+            "edge_mobile": {
+              "version_added": true
```
Comment (ExE-Boss) [inline api/CSS.json]: `false`

File: api/CSS.json [hunk 6]
```diff
@@ -0,0 +1,214 @@
...
+            "firefox": {
+              "version_added": "22",
+              "notes": "Was available behind the <code>layout.css.supports-rule.enabled</code> preference since Gecko 20."
+            },
+            "firefox_android": {
+              "version_added": "22",
+              "notes": "Was available behind the <code>layout.css.supports-rule.enabled</code> preference since Gecko 20."
+            },
+            "ie": {
+              "version_added": "6"
```
Comment (ExE-Boss) [inline api/CSS.json]: `false`

File: api/CSS.json [hunk 7]
```diff
@@ -0,0 +1,214 @@
...
+            },
+            "opera": {
+              "version_added": "12.1"
```
Comment (ExE-Boss) [inline api/CSS.json]: `52`

File: api/CSS.json [hunk 8]
```diff
@@ -0,0 +1,214 @@
...
+            },
+            "opera_android": {
+              "version_added": "12.1"
```
Comment (ExE-Boss) [inline api/CSS.json]: `52`

File: api/CSS.json [hunk 9]
```diff
@@ -0,0 +1,214 @@
...
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "22",
+              "notes": "Was available behind the <code>layout.css.supports-rule.enabled</code> preference since Gecko 20."
```
Comment (ExE-Boss) [inline api/CSS.json]: This should be done as:
```json
"firefox": [
  {
    "version_added": "22"
  },
  {
    "version_added": "20",
    "flags": [
      {
        "name": "layout.css.supports-rule.enabled",
        "type": "preference"
      }
    ]
  }
]
```

File: api/CSS.json [hunk 10]
```diff
@@ -0,0 +1,214 @@
...
+            },
+            "firefox_android": {
+              "version_added": "22",
+              "notes": "Was available behind the <code>layout.css.supports-rule.enabled</code> preference since Gecko 20."
```
Comment (ExE-Boss) [inline api/CSS.json]: This should be done as:
```json
"firefox_android": [
  {
    "version_added": "22"
  },
  {
    "version_added": "20",
    "flags": [
      {
        "name": "layout.css.supports-rule.enabled",
        "type": "preference"
      }
    ]
  }
]
```

File: api/CSS.json [hunk 11]
```diff
@@ -0,0 +1,214 @@
...
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": "22",
+            "notes": "Was available behind the <code>layout.css.supports-rule.enabled</code> preference since Gecko 20."
```
Comment (ExE-Boss) [inline api/CSS.json]: This should be done as:
```json
"firefox": [
  {
    "version_added": "22"
  },
  {
    "version_added": "20",
    "flags": [
      {
        "name": "layout.css.supports-rule.enabled",
        "type": "preference"
      }
    ]
  }
]
```

File: api/CSS.json [hunk 12]
```diff
@@ -0,0 +1,214 @@
...
+          },
+          "firefox_android": {
+            "version_added": "22",
+            "notes": "Was available behind the <code>layout.css.supports-rule.enabled</code> preference since Gecko 20."
```
Comment (ExE-Boss) [inline api/CSS.json]: This should be done as:
```json
"firefox_android": [
  {
    "version_added": "22"
  },
  {
    "version_added": "20",
    "flags": [
      {
        "name": "layout.css.supports-rule.enabled",
        "type": "preference"
      }
    ]
  }
]
```

File: api/CSS.json (hunk 3 above)
Comment (jpmedley) [reply api/CSS.json]: Actually neither. It landed in [February 2013](https://chromium.googlesource.com/chromium/src/+/1f311b68f9cee23d988e1d5a30cb3efab153e64a). That makes it older than 28, but I can't nail down the exact number. I would just put `true` and leave it at that.

File: api/CSS.json (hunk 8 above)
Comment (jpmedley) [reply api/CSS.json]: It's older than 52, but I can't say exactly how old.

File: api/CSS.json (hunk 3 above)
Comment (ExE-Boss) [reply api/CSS.json]: My comment was originally on line 83, but GitHub moved weirdly.

File: api/CSS.json (hunk 4 above)
Comment (ExE-Boss) [reply api/CSS.json]: My comment was originally on line 89, but GitHub moved weirdly.

File: api/CSS.json (hunk 5 above)
Comment (ExE-Boss) [reply api/CSS.json]: My comment was originally on line 92, but GitHub moved weirdly.

File: api/CSS.json (hunk 8 above)
Comment (ExE-Boss) [reply api/CSS.json]: My comment was originally on line 104, but GitHub moved weirdly.

**Pull request merged**